<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item alignment="Qt::AlignLeft">
     <widget class="QToolBox" name="settings">
      <widget class="QWidget" name="settingsPage1" native="true">
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <attribute name="label">
        <string/>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Avaliable Models</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox"/>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>195</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="filenameLabel">
          <property name="text">
           <string>Filename:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="listView">
          <property name="acceptDrops">
           <bool>true</bool>
          </property>
          <property name="dragEnabled">
           <bool>true</bool>
          </property>
          <property name="resizeMode">
           <enum>QListView::Fixed</enum>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="selectionRectVisible">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QWidget" name="chatWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Private and Offline PDF chatbot</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="chatDisplay">
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustIgnored</enum>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="InputWidget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="attatchFileLabel">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>../assets/attatch-file.png</normaloff>../assets/attatch-file.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="messageInput">
              <property name="inputMask">
               <string/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="clearButtonEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="sendButton">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>../assets/send-message.png</normaloff>../assets/send-message.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
